README.md

```diff
@@ -423,32 +416,76 @@ for obj in env.objects:
         # editing isn't supported yet!
 ```
 
-## Custom-Filesystem
+## Configurations
+
+There're several configurations and interfaces that provide the customizability to UnityPy.
+
+### Unity CN Decryption
+
+The Chinese version of Unity has its own builtin option to encrypt AssetBundles/BundleFiles. As it's a feature of Unity itself, and not a game specific protection, it is included in UnityPy as well.
+To enable encryption simply use the code as follow, with `key` being the value that the game that loads the bundles passes to `AssetBundle.SetAssetBundleDecryptKey`.
+
+```python
+import UnityPy
+UnityPy.set_assetbundle_decrypt_key(key)
+```
+
+### Unity Fallback Version
+
+In case UnityPy failed to detect the Unity version of the game assets, you can set a fallback version. e.g.
+
+```python
+import UnityPy.config
+UnityPy.config.FALLBACK_UNITY_VERSION = "2.5.0f5"
+```
+
+### Disable Typetree C-Implementation
+
+The [C-implementation](UnityPyBoost/) of typetree reader can boost the parsing of typetree by a lot. If you want to disable it and use pure Python reader, you can put the following 2 lines in your main file.
+
+```python
+from UnityPy.helpers import TypeTreeHelper
+TypeTreeHelper.read_typetree_boost = False
+```
+
+### Custom Block (De)compression
+
+Some game assets have non-standard compression/decompression algorithm applied on the block data. If you wants to customize the compression/decompression function, you can modify the corresponding function mapping. e.g.
+
+```python
+from UnityPy.enums.BundleFile import CompressionFlags
+flag = CompressionFlags.LZHAM
+
+from UnityPy.helpers import CompressionHelper
+CompressionHelper.COMPRESSION_MAP[flag] = some_function
```

Comment:
Instead of some or another it's probably better to simply name them

custom_compression_function and custom_decompression function,
with a stub above to show their expected functions signature.